src/pages/Search/SearchPage.tsx

```diff
@@ -120,15 +125,24 @@ function SearchPage({route}: SearchPageProps) {
     const queryJSON = useMemo(() => buildSearchQueryJSON(route.params.q), [route.params.q]);
     const {saveScrollOffset} = useContext(ScrollOffsetContext);
     const activeAdminPolicies = getActiveAdminWorkspaces(policies, currentUserPersonalDetails?.accountID.toString()).sort((a, b) => localeCompare(a.name || '', b.name || ''));
-
+    const {accountID} = useCurrentUserPersonalDetails();
+    const {defaultCardFeed} = useCardFeedsForDisplay();
+    const suggestedSearches = useMemo(() => getSuggestedSearches(accountID, defaultCardFeed?.id), [defaultCardFeed?.id, accountID]);
```

Comment:
My Chrome tab couldn't record a performance profile for me when working with this branch on the `exfy-perf@callstack.com` account (Applause workspace). I can't tell for sure, but might this be due to these calculations?

Reply:
FYI I had the same by only preloading the Search screen, no additional changes (for this account).